### PR TITLE
add file upload

### PR DESCRIPTION
**Pull request by jboretsky**
I think files are being uploaded properly. This was kind of a struggle cause documentation was a bit weird. The model thing in the form wasn't shown and i'm not 100% sure it's needed. Actual files are stored in integrichain/storage/..

### CODE REVIEW

**Comment by jboretsky** [comment]
it would also be nice if there was a way to download the files or something so that we can confirm it's working properly, but i'm not sure how to do that. The active storage docs are pretty limited

**Comment by jboretsky** [comment]
ok i confirmed it's working by putting an image link in, then uploading a jpg. The file shows up on the page so they are being stored correctly.

screenshot from 2018-03-17 22-30-09

**Comment by jboretsky** [comment]
also that is a very good chicken recipe

File: app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -11,11 +11,14 @@ def show
   end
 
   def new
+    @document = Document.new
     render :new
   end
 
   def create
-    @document = Document.create(document_params)
+    byebug
```
Comment (cpjk) [inline app/controllers/documents_controller.rb]: :fire: this debugger line

**Pull request merged**